mayhem/Mayhemfile

```diff
@@ -0,0 +1,12 @@
+# Namespaced project name that the target belongs to
```

Comment:
Can we remove the commented lines (especially image)? Also, we'll need a more descriptive binary name than `fuzz.libfuzzer`.